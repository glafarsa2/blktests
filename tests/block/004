#!/bin/bash
# SPDX-License-Identifier: GPL-3.0+
# Copyright (C) 2017 Jens Axboe
#
# Run a flush intensive workload

. tests/block/rc

DESCRIPTION="run lots of flushes"
TIMED=1

requires() {
	_have_fio
}

test_device() {
	echo "Running ${TEST_NAME}"

	FIO_PERF_FIELDS=("write iops")
	_fio_perf --bs=4k --rw=randwrite --norandommap --fsync=1 \
		--number_ios=256 --numjobs=64 --name=flushes \
		--filename="$TEST_DEV"

	echo "Test complete"
}
