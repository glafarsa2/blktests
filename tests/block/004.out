Running block/004
Test complete
